Localization/en-US.hjson

```diff
@@ -273,27 +272,27 @@ Mods: {
 			}
 
 			SoulHunt: {
-				Description: The enemy's defense is reduced
 				DisplayName: Soul Hunt
+				Description: The enemy's defense is reduced
 			}
 
 			BloodDrive: {
-				Description: Movement and attack speed are increased
 				DisplayName: Blood Drive
+				Description: Movement and attack speed are increased
 			}
 
 			PermafrostProtection: {
-				Description: Increased defense and rapid life regeneration
 				DisplayName: Permafrost Protection
+				Description: Increased defense and rapid life regeneration
 			}
 
 			ElectrifiedEnemy: {
-				Description: The enemy is taking damage over time
 				DisplayName: Electrified Enemy
+				Description: The enemy is taking damage over time
 			}
 
 			ChampionProtec: {
-				DisplayName: Champion's Protection
```

Comment:
'Champion's Protection' is the intended and correct name for the buff, despite the item being the 'Lost Hero's Shield', please revert that for at least the English localization.